survey-runner.tf

```diff
@@ -541,12 +615,26 @@ module "survey-runner-vpc" {
 }
 
 module "survey-runner-dynamodb" {
-  source                             = "github.com/ONSdigital/eq-terraform-dynamodb"
+  source                             = "github.com/ONSdigital/eq-terraform-dynamodb?ref=eq-2158-implement-dynamodb-with-rds-fallback"
   env                                = "${var.env}"
   aws_access_key                     = "${var.aws_access_key}"
   aws_secret_key                     = "${var.aws_secret_key}"
-  submitted_responses_read_capacity  = 1
-  submitted_responses_write_capacity = 1
+  submitted_responses_min_read_capacity  = 1
+  submitted_responses_max_read_capacity  = 500
+  submitted_responses_min_write_capacity = 1
+  submitted_responses_max_write_capacity = 500
+  questionnaire_state_min_read_capacity  = 1
+  questionnaire_state_max_read_capacity  = 500
+  questionnaire_state_min_write_capacity = 1
+  questionnaire_state_max_write_capacity = 500
+  eq_session_min_read_capacity  = 1
+  eq_session_max_read_capacity  = 500
+  eq_session_min_write_capacity = 1
+  eq_session_max_write_capacity = 500
+  used_jti_claim_min_read_capacity  = 1
+  used_jti_claim_max_read_capacity  = 500
+  used_jti_claim_min_write_capacity = 1
+  used_jti_claim_max_write_capacity = 500
```

Comment:
i would say 500 was a little high for dev default. maybe 25 max.
only really needs to be turned up for load testing which never normally happens on dev defaults